src/lib/VirtualScroller.js

```diff
@@ -289,6 +289,8 @@ class VirtualScroller {
             //  |--------------------|
             // newStartOffset    newEndOffset
             this.createItems(fragment, newStartOffset, newEndOffset);
+            // Delete all the current elements (if any)
+            this.deleteItems(this.listEl, 0);
```

Comment:
Minor, but it would be cool if calling `this.deleteItems(this.listEl)` cleared the items without the additional param.